src/applications/hca/config/chapters/veteranInformation/contactInformation.js

```diff
@@ -8,8 +8,7 @@ import { ContactInfoDescription } from '../../../components/FormDescriptions';
 import { isShortFormEligible } from '../../../utils/helpers';
 import { emptyObjectSchema } from '../../../definitions';
 
-const { email } = fullSchemaHca.properties;
-const { phone } = fullSchemaHca.definitions;
+const { email, phone } = fullSchemaHca.properties;
```

Comment:
Same thing here... `phone` isnt a property on its own now. Youll want to grab `homePhone` and `mobilePhone` as the properties for use.